README.md

```diff
@@ -1,5 +1,12 @@
 # Packaging with R
 
+## Teaching this lesson?
+Do you want to teach this lesson? This material is open-source and freely available. 
+Are you planning on using our material in your teaching? 
+We would love to help you prepare to teach the lesson and receive feedback on how it could be further improved, based on your experience in the workshop.
+
+You can notify us that you plan to teach this lesson by creating an issue in this repository. Also, it would be great if you can update [this overview of all workshops taught with this lesson material](workshops.md). This helps us show the impact of developing open-source lessons to our funders.
```

Comment:
```suggestion
You can notify us that you plan to teach this lesson by creating an issue in this repository. Also, it would be great if you could update [this overview of all workshops taught with this lesson material](workshops.md). This helps us show the impact of developing open-source lessons to our funders.
```